modules/standard/Regex.chpl

```diff
@@ -1243,60 +1277,91 @@ proc bytes.match(pattern: regex(bytes)):regexMatch
    to search for the pattern at any offset.
 
    :arg pattern: the compiled regular expression to match
-   :arg captures: (optional) what to capture from the regular expression. These
-                  should be strings or types that strings can cast to.
-   :returns: an :record:`regexMatch` object representing the offset in the
-             receiving string where a match occurred
+   :returns: true if string starts with :arg pattern:, false otherwise
```

Comment:
`:arg pattern:` is only useful at the start of a line, where it generates some formatting for it.  I think what you meant is ":arg:`pattern`" but I don't know that that will actually generate a link so it's probably best to surround `pattern` with two backtics.